### ISSUE TITLE
I cannot install tf-models-official on a PC with Windows 10. Error and undesired uninstallation.

### ISSUE THREAD

**Issue by anoldmaninthesea** [tensorflow/models#10270]
## 1. Describe the bug

I cannot install tf-models-official on a PC with Windows 10. Not only that, but because of the bug, it removes previously installed packages (Pandas, NumPy, etc.) !

## 3. Steps to reproduce

pip install tf-models-official on the command prompt, I get the following error

## 4. Additional context

Building wheels for collected packages: pycocotools
Building wheel for pycocotools (setup.py) ... error
ERROR: Command errored out with exit status 1:
command: 'C:\Users\iavta\anaconda3\envs\Ivo\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\iavta\\AppData\\Local\\Temp\\pip-install-f8z8t96a\\pycocotools_a6e2f889cdc04712b8898664a5be79ce\\setup.py'"'"'; __file__='"'"'C:\\Users\\iavta\\AppData\\Local\\Temp\\pip-install-f8z8t96a\\pycocotools_a6e2f889cdc04712b8898664a5be79ce\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d 'C:\Users\iavta\AppData\Local\Temp\pip-wheel-n4vlvzbc'
cwd: C:\Users\iavta\AppData\Local\Temp\pip-install-f8z8t96a\pycocotools_a6e2f889cdc04712b8898664a5be79ce\
Complete output (16 lines):
running bdist_wheel
running build
running build_py
creating build
creating build\lib.win-amd64-3.8
creating build\lib.win-amd64-3.8\pycocotools
copying pycocotools\coco.py -> build\lib.win-amd64-3.8\pycocotools
copying pycocotools\cocoeval.py -> build\lib.win-amd64-3.8\pycocotools
copying pycocotools\mask.py -> build\lib.win-amd64-3.8\pycocotools
copying pycocotools\__init__.py -> build\lib.win-amd64-3.8\pycocotools
running build_ext
cythoning pycocotools/_mask.pyx to pycocotools\_mask.c
C:\Users\iavta\anaconda3\envs\Ivo\lib\site-packages\Cython\Compiler\Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: C:\Users\iavta\AppData\Local\Temp\pip-install-f8z8t96a\pycocotools_a6e2f889cdc04712b8898664a5be79ce\pycocotools\_mask.pyx
tree = Parsing.p_module(s, pxd, full_module_name)
building 'pycocotools._mask' extension
error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools":
https://visualstudio.microsoft.com/visual-cpp-build-tools/
  ----------------------------------------
ERROR: Failed building wheel for pycocotools
Running setup.py clean for pycocotools
Failed to build pycocotools
Installing collected packages: six, tensorboard-data-server, pyparsing, numpy, grpcio, typing-extensions, threadpoolctl, text-unidecode, tensorflow-estimator, tensorboard, pywin32, python-dateutil, Pillow, kiwisolver, keras, joblib, httplib2, h5py, flatbuffers, cycler, colorama, clang, uritemplate, typeguard, tensorflow, tabulate, scikit-learn, regex, pytz, python-slugify, portalocker, matplotlib, google-auth-httplib2, google-api-core, dm-tree, tf-slim, tensorflow-text, tensorflow-model-optimization, tensorflow-addons, seqeval, sentencepiece, sacrebleu, pycocotools, py-cpuinfo, pandas, opencv-python-headless, oauth2client, kaggle, google-api-python-client, gin-config, tf-models-official
  Attempting uninstall: six
    Found existing installation: six 1.16.0
    Uninstalling six-1.16.0:
      Successfully uninstalled six-1.16.0
  Attempting uninstall: numpy
    Found existing installation: numpy 1.20.3
    Uninstalling numpy-1.20.3:
      Successfully uninstalled numpy-1.20.3
  Attempting uninstall: grpcio
    Found existing installation: grpcio 1.36.1
    Uninstalling grpcio-1.36.1:
      Successfully uninstalled grpcio-1.36.1
  Attempting uninstall: typing-extensions
    Found existing installation: typing-extensions 3.10.0.2
    Uninstalling typing-extensions-3.10.0.2:
      Successfully uninstalled typing-extensions-3.10.0.2
  Attempting uninstall: tensorflow-estimator
    Found existing installation: tensorflow-estimator 2.5.0
    Uninstalling tensorflow-estimator-2.5.0:
      Successfully uninstalled tensorflow-estimator-2.5.0
  Attempting uninstall: tensorboard
    Found existing installation: tensorboard 2.4.0
    Uninstalling tensorboard-2.4.0:
      Successfully uninstalled tensorboard-2.4.0
  Attempting uninstall: keras
    Found existing installation: Keras 2.4.3
    Uninstalling Keras-2.4.3:
      Successfully uninstalled Keras-2.4.3
  Attempting uninstall: h5py
    Found existing installation: h5py 2.10.0
    Uninstalling h5py-2.10.0:
      Successfully uninstalled h5py-2.10.0
  Attempting uninstall: tensorflow
    Found existing installation: tensorflow 2.3.0
    Uninstalling tensorflow-2.3.0:
      Successfully uninstalled tensorflow-2.3.0

## 6. System information

- Windows 10
- TensorFlow version 2.3.0
- Python 3.8
- Anaconda 2.0.4

**Comment by anoldmaninthesea** [comment]
https://docs.microsoft.com/en-us/answers/questions/136595/error-microsoft-visual-c-140-or-greater-is-require.html

It seems that we need to install the Compiler for C++. I solved this by downloading the full 6GB...
